pkg/vpcmodel/nodesExplainability.go

```diff
@@ -102,6 +104,21 @@ func (c *VPCConfig) getVsiNode(name string) Node {
 	return nil
 }
 
+// given a string address in non-cidr format return the list of all network interfaces attached to it
+func (c *VPCConfig) GetNetworkInterfaces(internalAddress string) (networkInterfaceNodes []Node, err error) {
+	if err != nil {
+		return nil, err
+	}
+
+	networkInterfaceNodes = []Node{}
+	for _, node := range c.Nodes {
+		if node.Kind() == resourceTypeNetworkInterface && node.Cidr() == internalAddress {
```

Comment:
@ShiriMoran do we really need to check `node.Kind()` and cannot just skip nodes that are `IsExternal()` ?

Reply:
What I had in mind was connecting specifically to network interfaces, assuming that internal nodes can represent other resources. But I'm also OK with returning all internal Nodes, and having more specific checks in other functionality, probably withing ibmvpc